README.md

```diff
@@ -44,3 +46,10 @@ optional arguments:
 ## Reference
 - This project is an improvement of [wifi-password](https://github.com/rauchg/wifi-password) created by [@rauchg](https://github.com/rauchg), where I have added support for multiple platforms and have added the feature for generating QR codes.
 - [wifiPassword](https://github.com/ankitjain28may/wifiPassword) created by [@ankitjain28may](https://github.com/ankitjain28may) was frequently used as reference throughout the development of this project.
+
+## Feature List
+
+(version 1.1.2)
+- Added fixes from the commit from [GoDoVoReZ:feature](https://github.com/GoDoVoReZ/wifi-password/tree/feature)
+- Added support for the Russian language of the system interface and cp866 encoding.
+- Added the ability to display SSID and password in terminal/console
```

Comment:
This is not needed. It can be seen in the commit history. It is also listed in the [release page](https://github.com/sdushantha/wifi-password/releases).